sdk/include/ndk/iofuncs.h

```diff
@@ -247,6 +247,7 @@ NtLoadDriver(
     _In_ PUNICODE_STRING DriverServiceName
 );
 
+__kernel_entry
```

Comment:
Would be nice if somehow this was part of the definition of `NTSYSCALLAPI`.

Reply:
I'm not sure. This would probably require including specstrings.h before defining it, and changing include order always lead to too much complication